src/locales/de/translation.json

```
@@ -182,6 +182,10 @@
     "accuracy": "Genauigkeit",
     "speed": "Schnelligkeit",
     "score-short": "Pkt",
-    "current-set": "Aktuelles Set"
+    "current-set": "Aktuelles Set",
+    "medal": "Hier könnte deine Medaille stehen!",
```

A little suggestion for this message: (To make it more gamified)
```suggestion
    "medal": "Leider hast du dieses Mal keine Medaille verdient!",
```